mne/decoding/time_delaying_ridge.py

```diff
@@ -8,56 +8,129 @@
 import warnings
 
 import numpy as np
+from scipy import sparse, linalg
 
 from .base import BaseEstimator
 from ..filter import next_fast_len
 from ..externals.six import string_types
 
 
 def _compute_corrs(X, y, smin, smax):
-    """Compute the auto- and cross-correlations."""
+    """Compute auto- and cross-correlations."""
```

Comment:
this relates to #3884

Reply:
Yep, and the changes here actually make it easier to use either pipeline interchangeably if we want to do so at some point